src/processor_arm.cpp

```diff
@@ -20,7 +20,10 @@
 #    include <sys/auxv.h>
 #  endif
 #endif
-
+#if defined _CPU_AARCH64_ && defined _OS_DARWIN_
```

Comment:
Not that important but I think you can merge this with the getauxval block above.